docs/website/references/cli/defradb_client_view_add.md

```
@@ -6,29 +6,38 @@ Add new view
 
 Add new database view.
 
-Use --lens-cid to specify a lens transform. Store a lens first using 'defradb client lens add'.
+Flag pairs <key>/<key>-file are mutually exclusive.
+
+Use --lens-cid to specify a lens transform. Store a lens first using 'defradb client lens add
 
 Learn more about the DefraDB GraphQL Schema Language on https://docs.source.network.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)